<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tInputDialog</class>
 <widget class="QWidget" name="tInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">TITLE</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string notr="true">Text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inputEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>okButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
